tests/daysOfTheWeek.js

```diff
@@ -15,5 +13,6 @@ tap.test('getDayName', (getDayNameTest) => {
   const validDateValue = validDateObject.getTime();
   getDayNameTest.equal(target.getDayName(validDateValue), 'Friday', 'get expected day name when using a Date timestamp');
   getDayNameTest.equal(target.getDayName(validDateObject), 'Friday', 'get expected day name when using a Date object');
+  getDayNameTest.equal(target.getDayName(), target.getDayName(new Date()), 'get expected day name when using no input');
   getDayNameTest.end();
```

Comment:
Excellent!  A test to make sure that calling `getDayname` with no inputs gives us the correct result.